src/jit/gentree.h

```diff
@@ -5077,7 +5077,7 @@ struct Statement
     GenTree*       m_rootTree;      // root of the expression tree
     GenTree*       m_treeList;      // first node (for forward walks)
     InlineContext* m_inlineContext; // The inline context for this statement.
-    IL_OFFSETX     gtStmtILoffsx;   // instr offset (if available)
+    IL_OFFSETX     m_ILoffsx;       // instr offset (if available)
```

Comment:
Any idea what the x suffix stands for? I'd just call it `m_ILoffset`.

Reply:
> Any idea what the x suffix stands for? I'd just call it m_ILoffset.

It is done to dittinguish `IL_OFFSETX` from `IL_OFFSET`. `IL_OFFSETX` has two bytes as flags (`IL_OFFSETX_STKBIT`, `IL_OFFSETX_CALLINSTRUCTIONBIT` and I am planning to add `compilerAdded` there soon) and special values like `ICorDebugInfo::NO_MAPPING`, `ICorDebugInfo::PROLOG`, `ICorDebugInfo::EPILOG`.